src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -2784,9 +2784,7 @@ private CorInfoType getChildType(CORINFO_CLASS_STRUCT_* clsHnd, CORINFO_CLASS_ST
                 result = asCorInfoType(returnType, clsRet);
             }
             else
```

Comment:
can you wrap it with { } because it looks a bit off

Reply:
Done.